modular_skyrat/modules/company_imports/code/armament_datums/kz_frontier.dm

```diff
@@ -0,0 +1,70 @@
+/datum/armament_entry/company_import/kz_frontier
+	category = KZ_EQUIPMENT_NAME
+	company_bitflag = CARGO_COMPANY_KZ_EQUIPMENT
+
+/datum/armament_entry/company_import/kz_frontier/manufacturing
+	subcategory = "Manufacturing Equipment"
+
+/datum/armament_entry/company_import/kz_frontier/manufacturing/rapid_construction_fabricator
+	item_type = /obj/item/flatpacked_machine
+	cost = CARGO_CRATE_VALUE * 8
+
+// Tools that you could use the rapid fabricator for, but you're too lazy to actually do that
+
+/datum/armament_entry/company_import/kz_frontier/basic
+	subcategory = "Hand-Held Equipment"
+	cost = PAYCHECK_COMMAND
+
+/datum/armament_entry/company_import/kz_frontier/basic/omni_drill
+	item_type = /obj/item/screwdriver/omni_drill
+
+/datum/armament_entry/company_import/kz_frontier/basic/prybar
+	item_type = /obj/item/crowbar/large/doorforcer
```

Comment:
I'm really not convinced that this is a good thing. I don't think having the ability to pry open doors should be available to anything outside of the jaws of life.

Reply:
See prior comment on the crowbar.